api/src/main/java/org/openmrs/module/eptsreports/reporting/library/converter/GenderDataConverter.java

```diff
@@ -0,0 +1,37 @@
+package org.openmrs.module.eptsreports.reporting.library.converter;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+
+public class GenderDataConverter implements DataConverter {
```

Comment:
The **PatientDataSetDefinition** that am using to display the columns need know how they can display the results, and the last argument would take a data converter object to guide that. I used that class to display gender the way it is required https://github.com/esaude/openmrs-module-eptsreports/blob/MEPTS-196_Data_Quality_Report/api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/EptsCommonUtils.java#L57. So if it is a requirement to change those values, it would be easly changed in the data converter

Reply:
Let me check if there is an already available means of displaying those in the API other than introducing that converter to handle the conversion

Reply:
Applied this **dsd.addColumn("Sex", new PersonToPatientDataDefinition(new GenderDataDefinition()), "");** instead of having a custom data converter